Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -26,6 +26,10 @@ public final class ReflectionService: CallHandlerProvider, Sendable {
     self.reflectionService.serviceName
   }
 
+  public init(protoFilePaths: [String]) throws {
```

Comment:
We aren't loading proto files (i.e. `.proto`) we're loading serialised file descriptor protos:

```swift
public init(serializedFileDescriptorProtoFilePaths: [String]) throws {
```

Reply:
This also needs documentation